.github/workflows/cmake.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
   test:
     strategy:
       matrix:
-        os: [ubuntu-24.04, macos-latest]
+        os: [ubuntu-24.04, macos-latest, windows-latest]
```

Comment:
We should probably add this to our `ci.yaml` file also right? There's also a line in there that checks for the runner os to equal Windows, but we don't have that os configured in the matrix.

Reply:
We don't support building with Bazel on Windows right now. We need to get cel-cpp compiling on Bazel on Windows before we can do that, but that's a bit more involved than with CMake.